File: CiderAndCode.Web/Controllers/PickApplesController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net;
using System.Net.Http;
using System.Web.Http;
using CiderAndCode.Web.DataModels;
using CiderAndCode.Web.ViewModels;

namespace CiderAndCode.Web.Controllers
{
    [RoutePrefix("PickApples")]
    public class PickApplesController : ApiController
    {
        [Route(""), HttpPost]
        public HttpResponseMessage PullApplesOffTree(ApplesPickedRequest request)
        {
            var db = new AppDbContext();

            var newBushel = new Bushel
            {
                Quantity = request.NumberOfBushels,
                User = db.Users.Find(request.UserId),
                Type = request.Type,
                Ripe = false
            };

            db.Bushels.Add(newBushel);
            db.SaveChanges();

            return Request.CreateResponse(HttpStatusCode.Created, newBushel);
        }

    }
}

File: CiderAndCode.Web/app/controllers/PickApples/pickApplesController.js
app.controller("pickApplesController",["$http","$scope",function($http, $scope) {
    $scope.bushel = {};
    $scope.users = [];

    $http.get("/api/users/list")
        .then(function(result) {
            $scope.users = result.data;
        });

    $scope.pickApples = function () {
        let bushel = $scope.bushel;
        $http.post("/api/PickApples",
                {
                    Type: bushel.appleType,
                    NumberOfBushels: bushel.numberOfBushels,
                    UserId: bushel.userId
                })
            .then(result => console.log(result))
            .catch(error => console.log(error));
    };

}]);